R/get_source_expressions.R

```diff
@@ -139,6 +139,41 @@ get_source_expressions <- function(filename, lines = NULL) {
         )
       }
 
+      parse_error_rx <- rex(
```

Comment:
add comments pointing to the parser source of these errors?

Reply:
Done.